webrender/res/ps_blend.glsl

```diff
@@ -4,13 +4,31 @@
 
 #include shared,prim_shared
 
+struct ColorTransforms {
+  mat4 matrix;
+  vec4 offset;
+};
+
+varying ColorTransforms vColorTransform;
```

Comment:
We've had issues in the past with driver bugs when a struct is passed as a varying, unfortunately. See https://github.com/servo/webrender/wiki/Driver-issues#1157---varying-structs for details. The workaround is to only pass GLSL primitive types as varyings.